packages/app/tailwind.config.js

```diff
@@ -109,6 +109,7 @@ module.exports = {
       },
       backgroundImage: {
         "matrix-gradient": "url('/assets/images/matrix-gradient-bg.png')",
+        "green-gradient": "linear-gradient(180deg, #E3FFCA 0%, #FFF 31.77%)",
```

Comment:
Now that you added another gradient, I couldn't tell the difference from the above.

So I think we could improve the name of `matrix-gradient` because it's a: gradient + grid/matrix lines.

Some ideas: `matrix-lines-with-gradient`, `matrix-grid-with-gradient`, `matrix-with-gradient`

Reply:
Named it `matrix-and-green-gradient`